tests/system/common.py

```diff
@@ -107,6 +107,10 @@ def command_health_check(command='true', failures=1, timeout=2):
         'maxConsecutiveFailures': failures
     }
 
+def unreachableStrategy(app_def, inactive=100, expunge=200):
```

Comment:
Hm, what is the motivation for this method if there is a `unreachable` fixture?

Reply:
used to created a variety of different unreachable configurations used by the test_unreachable.pyL65 for the parameterized testing.